config.toml

```diff
@@ -40,6 +40,18 @@
     # Make sure that this is greater than the unbonding period!
     SetGuardianEpochsDelay = 4 # 2 times the maximum unbonding period which can be 2 epochs - 1 round
 
+# ChainParametersByEpoch defines chain operation configurable values that can be modified based on epochs
+ChainParametersByEpoch = [
+    { EnableEpoch = 0, RoundDuration = 6000, ShardConsensusGroupSize = 21, ShardMinNumNodes = 58, MetachainConsensusGroupSize = 58, MetachainMinNumNodes = 58, Hysteresis = 0.2, Adaptivity = false },
+    { EnableEpoch = 9999999, RoundDuration = 6000, ShardConsensusGroupSize = 58, ShardMinNumNodes = 58, MetachainConsensusGroupSize = 58, MetachainMinNumNodes = 58, Hysteresis = 0.2, Adaptivity = false }
```

Comment:
enable epoch has to be set here also to `3677`

Reply:
fixed